content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Update and ship an Operator"
+date: 2021-03-10
+weight: 2
+description: > 
+    In depth explanation of operator update and shipment process. 
+--- 
```

Comment:
IHMO we need to;
- explain how to do by as a how to' using the operator bundle format\
- we need to explain the manual steps but also we can describe how sdk cli tool can help the users. I would indeed first explain via sdk and then manually.